torch/testing/_internal/common_methods_invocations.py

```diff
@@ -14404,6 +14430,9 @@ def error_inputs_mean(op_info, device, **kwargs):
         assert_autodiffed=False,
         supports_gradgrad=True,
         supports_out=False,
+        # test_reference_numerics only tests the case when the weight tensor is a scalar
+        sample_kwargs=sample_kwargs_prelu_scalar_weight,
```

Comment:
The sample inputs should also yield from the elementwise unary sample inputs, and a custom reference inputs should be added that does the same mutatis mutandi (yielding from the elementwise unary reference inputs)